### ISSUE TITLE
Raqm/libraqm not detected on Raspberry Pi

### ISSUE THREAD

**Issue by sevesalm** [python-pillow/Pillow#6023]
### What did you do?

I'm running Pillow on Raspberry Pi. I'm trying to use Raqm to use advanced font properties. Everything else works perfectly, but I have hard time getting Raqm to work. This probably has to do with installing the correct set of dependencies.

### What did you expect to happen?

I expected `PIL.features.get_supported_features()` to show that Raqm support is enabled. 

### What actually happened?

Raqm support was not listed/enabled.

### What are your OS, Python and Pillow versions?

* OS: Raspbian GNU/Linux 10 (buster)
* Python: Python 3.7.3
* Pillow:  8.4.0 installed with pip

Installed packages: 

`libfribidi0`
`libharfbuzz0b`
`libfreetype6`
`libraqm0`

Are there some other packages I need to install? Or am I missing some important configuration to make it work?

**Comment by nulano** [comment]
I think you should have the required packages. I realize you probably don't know the answer, but did pip install Pillow from a wheel or from source?

You could try running `python -m pip install -U --force-reinstall Pillow` and copying the full output to answer that question if reinstalling does not fix the issue.

**Comment by sevesalm** [comment]
Looks like wheel. Note that I specified 8.4.0 because for some reason 9.0.1 gave me an error (`AttributeError: module 'PIL' has no attribute 'features'`) when calling `PIL.features.get_supported_features()`. Anyway, the pip putput is similar:

```
# python -m pip install -U --force-reinstall Pillow==8.4.0

Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Collecting Pillow==8.4.0
  Using cached https://www.piwheels.org/simple/pillow/Pillow-8.4.0-cp37-cp37m-linux_armv7l.whl
Installing collected packages: Pillow
  Found existing installation: Pillow 8.4.0
    Uninstalling Pillow-8.4.0:
      Successfully uninstalled Pillow-8.4.0
Successfully installed Pillow-8.4.0
```

**Comment by nulano** [comment]
Ah, I would guess this is an issue with piwheels then.

Does installing from source work for you?
`python -m pip install -U --force-reinstall Pillow==8.4.0 --no-binary :all:`

**Comment by sevesalm** [comment]
@nulano, thanks for the tip! I had to install a few packages first (`python3-dev`, `libjpeg-dev`) and then I was able to install Pillow from the source. And it actually worked then, Raqm support is finally listed:

`['raqm', 'fribidi', 'harfbuzz', 'libjpeg_turbo']`

(And now I found out that the font I wanted to use with `lnum` feature turned on does not even support it 🤦🏼 But that is another issue.)

Anyway, thanks for the help and installing from the source was the solution to getting Raqm support working for Raspberry Pi. Closing this issue as resolved.
